incubation.md

```diff
@@ -0,0 +1,130 @@
+# Jupyter Incubation Process
+
+Incubation is the process by which certain types of new Jupyter Subprojects are created and
+evaluated for inclusion in the main Jupyter organization. In this context a Subproject is a
+GitHub repository with a development team that follows the Jupyter governance and contribution
+model.
+
+The incubation process doesn't apply to all new Subprojects, only those with the following
+characteristics:
+
+* Significant unanswered technical questions that required exploration.
```

Comment:
Good catches - fixes pushed.

On Fri, Sep 4, 2015 at 11:24 AM, Min RK <notifications@github.com> wrote:

> In incubation.md
> <https://github.com/jupyter/governance/pull/3#discussion_r38780338>:
>
> > @@ -0,0 +1,130 @@
> > +# Jupyter Incubation Process
> > +
> > +Incubation is the process by which certain types of new Jupyter Subprojects are created and
> > +evaluated for inclusion in the main Jupyter organization. In this context a Subproject is a
> > +GitHub repository with a development team that follows the Jupyter governance and contribution
> > +model.
> > +
> > +The incubation process doesn't apply to all new Subprojects, only those with the following
> > +characteristics:
> > +
> > +* Significant unanswered technical questions that required exploration.
>
> s/required/require/
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jupyter/governance/pull/3/files#r38780338>.
>



-- 
Brian E. Granger
Associate Professor of Physics and Data Science
Cal Poly State University, San Luis Obispo
@ellisonbg on Twitter and GitHub
bgranger@calpoly.edu and ellisonbg@gmail.com